governance/README.md

```diff
@@ -5,6 +5,7 @@ Security TAG is a [CNCF Technical Advisory Group](https://github.com/cncf/toc/tr
 * [Charter](charter.md) - mission and scope
 * [Roles](roles.md) - the work of the group is facilited by Chairs, Technical Leads, and active group members
 * [Process](process.md) - how projects are proposed and work is tracked
+* [Related Groups](./partner-groups/README.md) - List of CNCF, Kubernetes and other Industry wide groups that do related work 
```

Comment:
Does this location exist ? Do you mean `related-groups` ?

Reply:
Good catch! Fixed.